tests/backup/backup_helper.go

```diff
@@ -398,6 +398,44 @@ func CreateBackupWithoutCheck(backupName string, clusterName string, bLocation s
 	return nil
 }
 
+// CreateScheduleBackupWithoutCheck creates a schedule backup without waiting for success
+func CreateScheduleBackupWithoutCheck(scheduleName string, clusterName string, bLocation string, bLocationUID string,
```

Comment:
If you writing a function like this where there is no check happening, we need to return the response. Usually these kind of functions are called in negative cases where we expect it to fail and we need to validate the response. Please return the response here.

Reply:
CreateScheduleBackupWithoutCheck is similar to CreateBackupWithoutCheck, should I also include a return statement for the response in that CreateBackupWithoutCheck as well?

Reply:
Removed CreateScheduleBackupWithoutCheck.

Reply:
Yes

Reply:
Made CreateBackupWithoutCheck, CreateScheduleBackupWithoutCheck to return inspect response.